### PR TITLE
Merge 2.0 -> 2.1

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description (for Contributor and Core Developer)**

Following the merge of this PR https://github.com/akeneo/pim-community-dev/pull/7387

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Ok
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: upgrades/schema/Version_2_0_20171221083904_add_delete_cascade_on_variant_attribute_set.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Add a delete cascade to variant_attribute_set_has_hattributes on attribue_id
+ * Allow to delete an attribute contains in a variant family (not an axis)
+ */
+class Version_2_0_20171221083904_add_delete_cascade_on_variant_attribute_set extends AbstractMigration
```
Comment (ahocquard) [inline upgrades/schema/Version_2_0_20171221083904_add_delete_cascade_on_variant_attribute_set.php]: will it be launched when upgrading the 2.1?

File: src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php [hunk 2]
```diff
@@ -301,6 +301,10 @@ function it_normalizes_variant_product_updated_at_with_youngest_date_in_ancestor
 
 
         $product->getFamily()->willReturn($family);
+        $product->getFamily()->willReturn($family);
```
Comment (Doodoune) [inline src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php]: Double, delete one row

File: src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php [hunk 3]
```diff
@@ -321,22 +325,23 @@ function it_normalizes_variant_product_updated_at_with_youngest_date_in_ancestor
...
-                'id'            => '67',
-                'identifier'    => 'sku-001',
-                'created'       => $now->format('c'),
-                'updated'       => $now->format('c'),
-                'family' => [
+                'id'           => '67',
+                'identifier'   => 'sku-001',
+                'created'      => $now->format('c'),
+                'updated'      => $now->format('c'),
+                'family'       => [
```
Comment (ahocquard) [inline src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php]: don't bother with alignment next time (imo)

**Comment by SamirBoulil** [comment]
Ok guys, the CI is green so I'll merge but I'll make sure we settle the issue you raised.

**Pull request merged**